fastplotlib/graphics/line_collection.py

```diff
@@ -35,25 +35,27 @@ def __init__(
         Parameters
         ----------
         data: list of array-like
-            List of line data to plot, if elements 2D must be of shape [n_points, 2], if elements
-            3D must be of shape [n_points, 3]
+            List of line data to plot, each element must be a 1D, 2D, or 3D numpy array
```

Comment:
it's actually array or list of array-like.

Make a plot of temporal components and check which dimension corresponds to `n_lines`, it should be the first dimension. If not we need to change the implementation here in fastplotlib. For now we can allow array like and interpret it like `[n_lines, yvals]`

In the future we could allow more dimensional data for `LineCollection` like `[n_lines, xy]` but we can think about that later.

Reply:
can you confirm this behavior?

Reply:
@clewis7